applications_tests/dem_3d/periodic_boundary_gmsh.prm

```diff
@@ -0,0 +1,100 @@
+# Listing of Parameters
+# ---------------------
+
+#---------------------------------------------------
+# Simulation and IO Control
+#---------------------------------------------------
+subsection simulation control
+    set time step                = 1.5e-6
+    set time end       		     = 1.5e-6
+    set log frequency			 = 1
+    set output frequency         = 1
+end
+
+#---------------------------------------------------
+# Test
+#---------------------------------------------------
+subsection test
+    set enable 						 = true
+end
+
+# --------------------------------------------------
+# Model parameters
+#---------------------------------------------------
+subsection model parameters
+  set contact detection method 		   		 = dynamic
+  set load balance method				     = dynamic
+  set load balance threshold				 = 0.5
+  set dynamic load balance check frequency	 = 10000
```

Comment:
Based on your end time you are not doing any load balancing, so its better to remove the section if not using it.